src/main/java/de/orchound/gameoflife/game/Board.java

```diff
@@ -15,11 +16,17 @@ public Board(int width, int height) {
 		this.height = height;
 		source = new boolean[width * height];
 		target = new boolean[width * height];
+		randomize();
+	}
 
+	public void randomize() {
 		Random random = new Random();
 
 		for (int i = 0; i < target.length; i++) {
 			target[i] = random.nextBoolean();
+
+			// initial equals whatever target was, when it was randomized
+			initial = target.clone();
```

Comment:
There are two things wrong with this.
1. you probably intended to copy the array after randomizing the board, rather than every time you set the state of a cell.
2. `array.clone()` will create a new array which means it's gonna allocate memory, which could've already been allocated during the initialization of the application.

Instead of creating a new array here, initialize `initial` in the constructor as done for the other two board data arrays. Then reuse the array by copying the data from `target` over to `initial`.
See:
```java
System.arraycopy(target, 0, initial, 0, target.length);
```